certified-connectors/CyberProof/apiDefinition.swagger.json

```diff
@@ -170,7 +170,7 @@
 						"value-path": "pack_list",
 						"value-title": "pack_list",
 						"parameters": {
-							"category": "a_category"
+							"category": {"parameter":"a_category"}
```

Comment:
@sandeepkokare94 you have 'x-ms-dynamic-values' extension references properties within parameters. You need to define the 'x-ms-dynamic-list' extension. Please see our docs [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/openapi-extensions#how-to-use-dynamic-values)

Reply:
i tried using dynamic list , its take parameter reference , which is done on selection of drop down , i tried but not able to fetch the selected value from properties , 
so dynamic values takes a_category after selection of dropdown